draft-packages/popover/KaizenDraft/Popover/classMappers.ts

```diff
@@ -0,0 +1,118 @@
+import * as React from "react"
+import informativeIcon from "@kaizen/component-library/icons/information.icon.svg"
+import positiveIcon from "@kaizen/component-library/icons/success.icon.svg"
+import negativeIcon from "@kaizen/component-library/icons/exclamation.icon.svg"
+import styles from "./styles.scss"
+import { Side, Size, Variant, Position } from "./types"
+
+export const mapVariantToBoxClass = (variant: Variant): string => {
```

Comment:
These are all pretty much moved from the `Popover.tsx` file. The only exception being that I added the `Legacy` suffix where appropriate.